docs/infrastructure/deployment-targets/kubernetes-target/index.md

```diff
@@ -5,7 +5,7 @@ position: 50
 ---
 Kubernetes targets are used by the [Kubernetes steps](/docs/deployments/kubernetes/index.md) to define the context in which deployments and scripts are run.
 
-Conceptually, a Kubernetes target represent a permission boundary and an endpoint. Kubernetes [permissions](https://oc.to/KubernetesRBAC) and [quotas](https://oc.to/KubernetesQuotas) are defined against a namespace, and both the account and namespace are captured as a Kubernetes target, along with the cluster endpoint URL.
+Conceptually, a Kubernetes target represent a permission boundary and an endpoint. Kubernetes [permissions](https://oc.to/KubernetesRBAC) and [quotas](https://oc.to/KubernetesQuotas) are defined against a namespace, and both the account and namespace are captured as a Kubernetes target, along with the cluster endpoint URL.  The deployment target defines the default namespace, the namespace can be overwritten in the deployment process.
```

Comment:
I think this can use some of the other wording you had landed on:

> A namespace is required when registering the Kubernetes cluster with Octopus Deploy.  By default, the namespace used in the registration is used in health checks and deployments.  The namespace can be overwritten in the deployment process.

Reply:
Good shout, I've updated it.